src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,141 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static class Node<T> {
+        private T element;
+        private Node<T> next;
+
+        public Node(T element) {
```

Comment:
constructor shold initialize all fields (see common mistakes)

Reply:
Not resolved